### PR TITLE
NO-DED Linking new versions of the methods in Conf v2

### PR DESCRIPTION
**Pull request by kamilatokarska**
# Deploy preview

https://63eba54f12738a12e816a6f2--livechat-public-docs.netlify.app/management/configuration-api/v2.0#agents-deprecated

# Links

Link your Jira ticket.

- [Jira](https://livechatinc.atlassian.net/browse/DED-524?atlOrigin=eyJpIjoiZjFiZjRkZDBlYmJkNGY0ODk1MmM4MzhmM2MwZjBjNTQiLCJwIjoiaiJ9)

# Description

Links added. I have done it the way some methods were already described in this document (put info in the yellow box). Please check if the links added are the correct ones.

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

**Review by ale-kacperczyk — CHANGES_REQUESTED** [review]
Two general remarks:

1. The article 'the' is missing in some of the descriptions.
2. It'd be better if the new method's name was implicitly written in the copy in case one of the links is broken and someone has to search for it manually.

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 1]
```diff
@@ -864,7 +900,7 @@ curl "https://api.livechatinc.com/chats?\
 
 <Warning>
 
-**Please note this method is deprecated**. We highly recommend switching to the new [Messaging APIs](/messaging/).
+We highly recommend switching to the new [Get Chat](messaging/agent-chat-api#get-chat) method.
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:903]:
```suggestion
We highly recommend switching to the new [Get Chat](/messaging/agent-chat-api#get-chat) method.
```

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 2]
```diff
@@ -2997,7 +3045,13 @@ curl "https://api.livechatinc.com/groups" \
 
 <Text>
 
-## Get a single group details
+## Get a single group details (deprecated)
+
+<Warning>
+
+We highly recommend switching to the new [Get Group](/management/configuration-api#get-group) method.
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:3052]: FYI, I have fixed the link but haven't generated the new preview

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 3]
```diff
@@ -864,7 +900,7 @@ curl "https://api.livechatinc.com/chats?\
 
 <Warning>
 
-**Please note this method is deprecated**. We highly recommend switching to the new [Messaging APIs](/messaging/).
+We highly recommend switching to the new [Get Chat](/messaging/agent-chat-api#get-chat) method.
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:903]: FYI, I have fixed the link but haven't generated the new preview

**Comment by kamilatokarska** [comment]
> Two general remarks:
> 
> 1. The article 'the' is missing in some of the descriptions.
> 2. It'd be better if the new method's name was implicitly written in the copy in case one of the links is broken, and someone has to search for it manually.

Ad.1 - I have changed the copy of the warnings I was adding, so now `the` should be fine
Ad.2 - all methods have `deprecated` added in their names and specific names given in the warnings

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 4]
```diff
@@ -154,7 +154,13 @@ You can call the LiveChat API using librariers written in backend languages. To
   Did you write your own library and want it listed here? Let us know!
 </aside>
 
-# Agents
+# Agents (deprecated)
+
+<Warning>
+
+**Please note this API is deprecated**. We highly recommend switching to the new [Management APIs](/management/).
```
Comment (ale-kacperczyk) [inline src/pages/management/configuration-api/v2.0/index.mdx:161]: Wouldn't this be a better place to link to? `/management/configuration-api#agents`

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 5]
```diff
@@ -3332,7 +3416,13 @@ curl "https://api.livechatinc.com/status/1" \
 
 </Section>
 
-# Tags
+# Tags (deprecated)
+
+<Warning>
+
+**Please note this API is deprecated**. We highly recommend switching to the new [Management APIs](/management/).
```
Comment (ale-kacperczyk) [inline src/pages/management/configuration-api/v2.0/index.mdx:3423]: `management/configuration-api#tags`

File: src/pages/management/configuration-api/v2.0/index.mdx (hunk 4 above)
Comment (kamilatokarska) [reply src/pages/management/configuration-api/v2.0/index.mdx:161]: You're right :)

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 6]
```diff
@@ -3276,7 +3348,13 @@ curl "https://api.livechatinc.com/groups/4" \
 
 </Section>
 
-# Status
+# Status (deprecated)
+
+<Warning>
+
+**Please note this API is deprecated**. We highly recommend switching to the new [Messaging APIs](/messaging/).
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:3355]:
```suggestion
**Please note this API is deprecated**. We highly recommend switching to the new [Messaging APIs](/messaging/agent-chat-api#status).
```

File: src/pages/management/configuration-api/v2.0/index.mdx (hunk 5 above)
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:3423]:
```suggestion
**Please note this API is deprecated**. We highly recommend switching to the new [Management APIs](/management/configuration-api#tags).
```

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 7]
```diff
@@ -2905,6 +2941,12 @@ curl "https://api.livechatinc.com/greetings/2491" \
 
 # Groups
 
+<Warning>
+
+**Please note this API is deprecated**. We highly recommend switching to the new [Management APIs](/management/).
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:2946]:
```suggestion
**Please note this API is deprecated**. We highly recommend switching to the new [Management APIs](/management/configuration-api#groups).
```

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 8]
```diff
@@ -2905,6 +2941,12 @@ curl "https://api.livechatinc.com/greetings/2491" \
 
 # Groups
 
+<Warning>
+
+**Please note this API is deprecated**. We highly recommend switching to the new [Management APIs](/management/configuration-api#groups).
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:2946]: The link has been corrected but the new preview with it has not been generated

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 9]
```diff
@@ -3557,7 +3665,13 @@ curl "https://api.livechatinc.com/tags/support" \
 
 </Section>
 
-# Tickets
+# Tickets (deprecated)
+
+<Warning>
+
+**Please note this API is doesn't have its newer equivalent in LiveChat**. We highly recommend migrating to <a href="https://www.helpdesk.com/" target="_blank">HelpDesk</a>. HelpDesk API Documentation can be found <a href="https://api.helpdesk.com/docs" target="_blank">here</a>.
```
Comment (litwinow) [inline src/pages/management/configuration-api/v2.0/index.mdx:3672]:
```suggestion
**Please note this API doesn't have its newer equivalent in LiveChat**. We highly recommend migrating to <a href="https://www.helpdesk.com/" target="_blank">HelpDesk</a>. HelpDesk API Documentation can be found <a href="https://api.helpdesk.com/docs" target="_blank">here</a>.
```

**Review by litwinow — APPROVED** [review]

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**